autofill-parser/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 
 - [Mull](https://divestos.org/pages/our_apps#mull) is now supported as an Autofill-capable browser when installed from the divestos.org [repo](https://divestos.org/fdroid/official/).
 - [Cromite](https://github.com/uazo/cromite) is now supported as an Autofill-capable browser.
+- Calyx's [Chromium](https://github.com/uazo/cromite) is now supported as an Autofill-capable browser.
```

Comment:
```suggestion
- Calyx's [Chromium](https://github.com/uazo/cromite) is now supported as an Autofill-capable browser.
```
Is the link right?

Reply:
No, i copied it on accident and forgot to fill it in.

I am being specific about Calyx here because the base64 key may be different from that of the official build, as CalyxOS distrubutes a patched version of Chromium.

Should i keep it as is or remove the additional information?

Reply:
It's fine to call it Calyx's Chromium as long as there is a link that points to a source code repository or binary downloads.